pkg/authorization/authorizer/scope/converter.go

```diff
@@ -210,10 +211,18 @@ func (userEvaluator) ResolveGettableNamespaces(scope string, clusterPolicyGetter
 // escalatingScopeResources are resources that are considered escalating for scope evaluation
 var escalatingScopeResources = []unversioned.GroupResource{
```

Comment:
It is still wrong - you are missing legacy for `oauthauthorizetokens` and new for `oauthaccesstokens`.